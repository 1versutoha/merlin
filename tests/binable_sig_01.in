["find", "use", "bin_prot"]
["tell", "source", "module T : sig type t with bin_io end = struct type t = T with bin_io end\nmodule T1 : Bin_prot.Binable.S = T"]
["tell", "end"]
["errors"]
