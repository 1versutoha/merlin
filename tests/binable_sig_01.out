["return",true]
["return",null]
["return",{"line":2,"col":34}]
["return",[]]
